apps/damap-frontend/src/app/components/layout/layout.component.html

```diff
@@ -98,9 +98,20 @@
 </mat-sidenav-container>
 <div class="footer">
   <div class="version-information">
-    {{ "layout.version" | translate }}: {{ version }}
-    <a [routerLink]="'/gdpr'" rel="noopener">{{ "layout.gdpr" | translate }}</a>
+    <div>
+      {{ "layout.version" | translate }}: {{ version }}
+      <a [routerLink]="'/gdpr'" rel="noopener">{{
+        "layout.gdpr" | translate
+      }}</a>
+    </div>
+    <a
+      class="mat-focus-indicator"
+      [href]="'layout.logo.link' | translate"
+      rel="noopener">
+      <img src="assets/damap_white_new.png" alt="Logo" height="30" />
```

Comment:
minor: could be a bit taller as it is difficult to read the text beneath the logo